src/Shouldly.Shared/DiffToolConfiguration.cs

```diff
@@ -0,0 +1,98 @@
+#if !PORTABLE
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Shouldly
+{
+    public class DiffToolConfiguration
+    {
+        readonly List<IShouldNotLaunchDiffTool> _knownShouldNotLaunchDiffToolReasons;
+        readonly List<DiffTool> _diffToolPriority = new List<DiffTool>();
+        readonly List<DiffTool> _diffTools;
+
+        public static class KnownDiffTools
+        {
+            public static readonly DiffTool KDiff3 = new DiffTool("KDiff3", @"C:\Program Files\KDiff3\kdiff3.exe", (received, approved) => $"\"{received}\" \"{approved}\" -o \"{approved}\"");
+        }
+
+        public static class KnownDoNotLaunchStrategies
+        {
+            public static readonly IShouldNotLaunchDiffTool NCrunch = new DoNotLaunchEnvVariable("NCRUNCH");
```

Comment:
When you run an individual test in ncrunch does this environment variable still exist?